docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    We also have some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of
+    newcomer to developer.
```

Comment:
"**a**" newcomer and add a line break in the body as the first line exceeds 70 characters.

Reply:
sorry the gitmate comments were for earlier commits

Reply:
Sorry, I did not understand your comment. Is this PR correct now?

Reply:
just add "a" before newcomer and rest is fine

Reply:
commit body mention~~s~~